### PR TITLE
tracking protection settings

### PR DESCRIPTION
**Pull request by chung-lai**
Add tracking protection settings to Cookie cutter settings

Screen Shot 2022-05-17 at 9 30 56 AM

### How was this tested?
- [x] I tested this manually
- [ ] I added automated tests
- [ ] Existing automated tests are enough to cover my changes

### CODE REVIEW

**Comment by chung-lai** [comment]
@hwiatneeva just putting up something for us to iterate on. We are thinking to enable cookie cutter in upcoming release.

@evancrow fyi

File: Client/Frontend/Overlay/Content/Menu/TrackingProtection/TrackingMenuSettingsView.swift [hunk 1]
```diff
@@ -78,22 +51,34 @@ struct TrackingSettingsBlock: View {
...
+                    VStack(alignment: .leading) {
+                        Text(strength.name.capitalized)
+                            .padding(.bottom, -4)
+                        Text(strength.description).withFont(.bodySmall).foregroundColor(
+                            .secondaryLabel
+                        ).frame(width: geometrySize.width - 120, alignment: .leading)
+                    }
                     .tag(strength.rawValue)
+                }
+                .padding(.top, 5)
```
Comment (evancrow) [inline Client/Frontend/Overlay/Content/Menu/TrackingProtection/TrackingMenuSettingsView.swift:76]: Can make this one line:

```suggestion
                .padding(.vertical, 5)
```

File: Client/Frontend/Overlay/Content/Menu/TrackingProtection/TrackingMenuSettingsView.swift [hunk 2]
```diff
@@ -78,22 +51,34 @@ struct TrackingSettingsBlock: View {
...
+            )
+        ) {
+            Picker("Protection Mode", selection: $contentBlockingStrength) {
+                ForEach(BlockingStrength.allCases) { strength in
+                    VStack(alignment: .leading) {
+                        Text(strength.name.capitalized)
+                            .padding(.bottom, -4)
+                        Text(strength.description).withFont(.bodySmall).foregroundColor(
+                            .secondaryLabel
+                        ).frame(width: geometrySize.width - 120, alignment: .leading)
```
Comment (evancrow) [inline Client/Frontend/Overlay/Content/Menu/TrackingProtection/TrackingMenuSettingsView.swift:72]: Where did this 120 come from?

**Review by hwiatneeva — COMMENTED** [review]
**Note: the following is QA finds from Neeva 1.41.2 (222) -- might be older than the latest PR.**

===================
Overall lovely!

Launch blocking issues... they're pixel nits:

1. Ship ‘Ad Blocker’ laterrrr (hide from this launch)

2. String for tracking protection options:
‘Block Cookies’ → Standard
‘Block Cookies (Strict)’ → Strict

3. ‘COOKIE NOTICES’ → COOKIE POPUPS

4. ‘TRACKING PROTECTION’ → ‘TRACKERS’
image

image

Here's a [QA deck](https://docs.google.com/presentation/d/1pvPjEqSAW7Oba2p6ceV2yL5bUiQsnNrpiZppjiWo75s/edit#slide=id.g12a56f22104_0_10). Side process note: this deck is just a try to assess if this is useful, and repeated doable.

Thanks much, @evancrow & @chung-lai  !!!

**Comment by hwiatneeva** [comment]
**Feature Requests for future revisions:**

1. Settings for Trackers –  ideas: more familiar copy and additional explanation
2. Settings for education – Ideas: short videos for each feature
3. Tracker section can be flattened with ‘Standard’ and ‘Strict’ options in the first screen (and remove the second screen). Also the switch might be unnecessary. 

image
image

File: Client/Frontend/Overlay/Content/Menu/TrackingProtection/TrackingMenuSettingsView.swift (hunk 2 above)
Comment (chung-lai) [reply Client/Frontend/Overlay/Content/Menu/TrackingProtection/TrackingMenuSettingsView.swift:72]: Magic number, will define it somewhere as constant, it's basically the checkmark width

**Comment by chung-lai** [comment]
> **Note: the following is QA finds from Neeva 1.41.2 (222) -- might be older than the latest PR.**
> 
> =================== Overall lovely!
> 
> Launch blocking issues... they're pixel nits:
> 
> 1. Ship ‘Ad Blocker’ laterrrr (hide from this launch)
> 2. String for tracking protection options:
>    ‘Block Cookies’ → Standard
>    ‘Block Cookies (Strict)’ → Strict
> 3. ‘COOKIE NOTICES’ → COOKIE POPUPS
> 4. ‘TRACKING PROTECTION’ → ‘TRACKERS’
>    image
> 
> image
> 
> Here's a [QA deck](https://docs.google.com/presentation/d/1pvPjEqSAW7Oba2p6ceV2yL5bUiQsnNrpiZppjiWo75s/edit#slide=id.g12a56f22104_0_10). Side process note: this deck is just a try to assess if this is useful, and repeated doable.
> 
> Thanks much, @evancrow & @chung-lai !!!

Thanks @hwiatneeva, updated the screen with the launch blocking changes and also flatten the list without the second screen. PTAL

**Review by hwiatneeva — COMMENTED** [review]
LGTM from design. Thanks!

**Review by evancrow — APPROVED** [review]
LGTM!

**Comment by darinwf** [comment]
Removing `cherry-pick-needed` since we decided to not pull cookie cutter into this week's release.

**Pull request merged**